vpr/src/base/check_netlist.cpp

```diff
@@ -93,8 +93,9 @@ static int check_connections_to_global_clb_pins(ClusterNetId net_id, int verbosi
         ClusterBlockId blk_id = cluster_ctx.clb_nlist.pin_block(pin_id);
         int pin_index = cluster_ctx.clb_nlist.pin_physical_index(pin_id);
 
-        if (physical_tile_type(blk_id)->is_ignored_pin[pin_index] != net_is_ignored
-            && !is_io_type(physical_tile_type(blk_id))) {
+        auto logical_type = cluster_ctx.clb_nlist.block_type(blk_id);
+        if (physical_tile_type(logical_type)->is_ignored_pin[pin_index] != net_is_ignored
```

Comment:
physical_tile_type may return nullptr

Reply:
Also not sure this logic is valid

Reply:
This check should only be done on a placed netlist anyways?  Or you need to check all possible physical placements?

Reply:
Another possibility here is to have "is_ignored_pin" be a property of the physical tile, but enforce that equivilance requires that all physical<->logical types have the same "is_ignored_pin" property for ports.  Then this check could safely work as "use any physical tile for this logical tile".

Reply:
Yeah, I prefer the second option actually. During the loading of the data structures I can make sure that all `logical_blocks` of a `physical_tile` do have the same pin properties.

Reply:
Done